docs/openj9_support.md

```diff
@@ -102,13 +168,43 @@ When public support for an operating system version ends, OpenJ9 can no longer b
 | AIX POWER BE 64-bit         | AIX 7.1 TL04             | xlc/C++ 13.1.3                  |                |
 
 
-OpenJDK9 binaries are supported on the minimum operating system levels shown in the table.
-For Linux, other distributions are supported provided that they have a Linux kernel v4.4 or later
-(Look in the `/proc/version` file to find the kernel version number).
+### OpenJDK 10
+
+OpenJDK8 binaries are supported on the minimum operating system levels shown in the following tables:
```

Comment:
Says OpenJDK8 but this is the section for 10.

Reply:
Fixed